services/IDonor.cs

```diff
@@ -0,0 +1,37 @@
+using Stateless;
+
+namespace NosAyudamos
+{
+    public interface IDonor
+    {
+        State State { get; set; }
+        void Register();
+    }
+
+    public class Donor : IDonor
+    {
+        private readonly StateMachine<State, Trigger> _machine;
```

Comment:
No underscore prefix plz. We need a .editorconfig that will ensure some conventions for all of us :)